crates/driver/src/tests/setup/mod.rs

```diff
@@ -1517,23 +1556,21 @@ pub enum SettleStatus {
     },
 }
 
-pub struct SettleOk<'a> {
-    test: &'a Test,
```

Comment:
Removing the lifetime param is a workaround to fix the compilation issue. More details: https://cowservices.slack.com/archives/C0375NV72SC/p1733156004100299